code/modules/admin/verbs/SDQL2/SDQL_2_parser.dm

```diff
@@ -396,7 +396,7 @@
 	string(i, list/node)
 
 		if(copytext(token(i), 1, 2) in list("'", "\""))
-			node += token(i)
+			node += copytext(token(i),2,-1)
```

Comment:
Probably, but common usage is func("Stuff") or value = "Stuff" and that makes that easier.